guides/common/modules/proc_enabling-salt-grain-uploads.adoc

```diff
@@ -7,15 +7,17 @@ Salt Grains are collected system properties, for example the operating system or
 .Procedure
 * On your Salt Master, edit `/etc/salt/foreman.yaml`:
 +
-[options="nowrap" subs="attributes"]
+[source, yaml, options="nowrap", subs="verbatim,quotes,attributes"]
 ----
 :proto: https
-:host: {foreman-example-com}
+:host: __{smartproxy-example-com}__
```

Comment:
I think, this should stay the URL of your Foreman instance.

Reply:
@nadjaheitmann is correct that this must remain `{foreman-example-com}`. There is no Foreman API on port 443. This is similar to how the Puppetserver works: it talks back directly to Foreman.